src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -4,50 +4,50 @@
     "origin.toomanyorigins.hare.name": "Hare",
 
     "origin.toomanyorigins.hisskin.name": "Hiss-kin",
-    "origin.toomanyorigins.hisskin.description": "Creeperın bu uzak akrabası, patlamalarını kaçış amaçlı kullanır.",
+    "origin.toomanyorigins.hisskin.description": "Creeper'ın akrabası olan Hiss-kin, patlamalarını kaçış amaçlı kullanır.",
```

Comment:
added the ' back because thats how it is in the translation for Blazeborn's description
This now directly translates to "Hiss-kin, whom is a relative of Creeper, *"